testing/xfce4-sensors-plugin/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: Carlo Landmeter <clandmeter@gmail.com>
+# Maintainer:
```

Comment:
I don't want to take over maintainership.
If it's not acceptable to bring a packages into /testing without a maintainer then I can close this issue.